.github/workflows/build.yaml

```diff
@@ -60,19 +60,25 @@ jobs:
       # those targets first and only builds the package if they succeed.
       # Build the sdist and wheel distribution of the package and docs as a zip file.
       run: make dist
+    # Generate the requirements.txt that contains the hash digests of the dependencies and
+    # generate the SBOM using CyclonDX SBOM generator.
+    - name: Generate requirements and SBOM for all packages
+      run: make requirements sbom
+    # Audit all required and currently installed packages for security vulnerabilities.
+    - name: Audit required packages
+      run: make audit
+    # Remove the old requirements.txt file (which includes _all_ packages) and generate a
+    # new one for the package and its actual and required dependencies only.
+    - name: Prune requirements for shipping package
+      run: |
+        rm requirements.txt
+        make prune requirements
```

Comment:
Should we add a
```yaml
if: matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON
```
to these three steps?

Reply:
As discussed, let’s run audit before the SBOM: commit https://github.com/jenstroeger/python-package-template/pull/307/commits/5b721256bdf5f7531af4f88fd488765afa5757d7. Keeps this change clean and simple 👍🏼